core/ViewDataTable/Manager.php

```diff
@@ -184,7 +184,7 @@ public static function configureFooterIcons(ViewDataTable $view)
         $nonCoreVisualizations = static::getNonCoreViewDataTables();
 
         foreach ($nonCoreVisualizations as $id => $klass) {
-            if ($klass::canDisplayViewDataTable($view)) {
+            if ($klass::canDisplayViewDataTable($view) || $view->isViewDataTableId($id)) {
```

Comment:
I haven't tested... does this somehow affect inherited tables as well? Like GoalTable is selected, and it would make regular htmltable or others maybe available accidentally?

Reply:
AFAIK `isViewDataTableId` also is true for any inherited.

Reply:
changed the implementation to an direct compare